config/katello.migrations/190904144224-set_tuning_fact.rb

```diff
@@ -0,0 +1 @@
+scenario[:facts]['tuning'] ||= 'small'
```

Comment:
Based on the test failures, this needs to check that `scenario[:facts]` exists prior to trying to set a property of it. Also, Ruby makes me sad sometimes with the mixing of symbols and strings.